### ISSUE TITLE
Interpolation for vector fields

### ISSUE THREAD

**Issue by ggebbie** [JuliaClimate/MeshArrays.jl#62]
My goal is to take staggered vector data from an LLC grid (ECCO v4r4) to produce NVEL and EVEL fields on a regular grid for plotting purposes. In concept, I'd like to follow the steps in https://github.com/ECCO-GROUP/ECCOv4-py/blob/master/ecco_v4_py/vector_calc.py.  I see that the Vector Fields section of the doc is in the works, but I couldn't tell if the src code already can handle vector interpolation. I investigated `InterpolationFactors` and `gcmvector` type, but my guess is that they don't already do what I need. Could you give a quick status update regarding this issue?

**Comment by gaelforget** [comment]
Yes, this is something that needs to be in the docs. There is an example in the [Julia Climate notebook folder](https://github.com/JuliaClimate/GlobalOceanNotebooks). See the bottom part of  https://nbviewer.org/github/JuliaClimate/GlobalOceanNotebooks/blob/master/OceanTransports/04_transports.ipynb

**Comment by gaelforget** [comment]
Docs have now been updated. Please refer to the new vector field tutorial (currently @ https://juliaclimate.github.io/MeshArrays.jl/dev/start/vectors.html). I plan to consolidate a few others things over next week or so (e.g. more examples for `isosurface` and `curl`) but  new release `v0.2.27` should have all needed functions. Closing this issue but dont hesitate to reopen if needed.

**Comment by ggebbie** [comment]
I confirm that I was able to do interpolation for vector fields using information that you've already provided. I rewrote some of the scripts available in JuliaClimateNotebooks under Ocean Transport. Now, I should compare my `ECCOtour.rotate_velocity!` function to the one that you have written. It will be especially useful to compare performance because I found this step to be a bottleneck in my code.

The new documentation looks fantastic. Some of the un-rotated velocity plots look very familiar!
